docs/options.md

```diff
@@ -123,4 +123,59 @@ into this:
 }]
 ```
 
+This can also be used with raw sql queries, using the .query interface. Note that options need to be passed as the third argument to .query, as the second argument is used for params.
+
+```javascript
+db.query(
+  'select u.id as u_id, u.name as u_name, u.address as u_address,  t.id as t_id, t.score as t_score ' +
+  'from users u inner join tests t ' +
+  'on t.user_id = u.id', [], {
+    decompose: {
+      pk: 'id',
+      columns: {u_id: 'id', u_name: 'name', u_address: 'address'},
+      tests: {
+        pk: 't_id',
+        columns: {t_id: 'id', t_score: 'score},
```

Comment:
you've got a missing apostrophe here which is messing up the format. I think the `db.query` statement is sufficient, though -- the effect is already demonstrated with the schema above so there's no need to duplicate the sample input/output here too.